src/lib/arch/aarch64/Instruction_execute.cc

```diff
@@ -486,6 +520,66 @@ void Instruction::execute() {
         branchAddress_ = instructionAddress_ + metadata_.operands[0].imm;
         break;
       }
+      case Opcode::AArch64_BF16DOTlanev8bf16: {  // bfdot vd.4s, vn.8h,
```

Comment:
Do the bf16 instructions have test cases?

Reply:
No, I've kept them as experimental implementations and undocumented.